Week-4/Session 9/Session-09-1/script.js

```diff
@@ -0,0 +1,20 @@
+const divideArray = (nums) => {
+  let evenNums = [];
+  let oddNums = [];
+  if (nums.length > 0) {
+    nums.forEach((element) => {
+      if (element % 2 === 0) evenNums.push(element);
+      else oddNums.push(element);
+    });
+    evenNums.sort((a, b) => a - b);
+    oddNums.sort((a, b) => a - b);
```

Comment:
array.sort() will do the same thing. 
Please note it will change the original array and not return a new array.